cellxgene_resources/curation_sample_code.ipynb

```diff
@@ -800,16 +785,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Extract and de-duplicate gene annotations and write them to a tsv file\n",
-    "dir = ''\n",
-    "gtf = ''\n",
-    "\n",
-    "gtf_df = extract_gene_id_and_symb(dir, gtf)\n",
-    "gene_id_symbol = assign_multiple(gtf_df)\n",
-    "\n",
-    "# write dataframe to tsv file in ref_file folder\n",
-    "tsv_file = gtf.replace('.','_') + '.tsv' \n",
-    "gene_id_symbol.to_csv('ref_files/' + tsv_file, sep=\"\\t\")"
+    "# Write gene_df to tsv file\n",
+    "dir = 'path/to/working/dir'\n",
+    "tsv_file = ref_file.split('/')[4].replace('.','_') + '.tsv'\n",
+    "gene_df.to_csv(dir + tsv_file, sep=\"\\t\")"
```

Comment:
Following above suggestions:

- the `dir = ` line is removed as it is addressed above
- the tsv_file naming gets much simpler (`ref_file.replace('.','_') + '.tsv'`)
- update the `dir` --> `my_dir`
- in the `sep`, I'd use single quotes for consistency with the rest of the code
- add `index=False` to the `to_csv()`